src/lib/src/select-container.component.ts

```diff
@@ -238,6 +238,7 @@ export class SelectContainerComponent implements AfterViewInit, OnDestroy {
 
   private observeBoundingRectChanges() {
     // Initialize client bounding rect
+    this.calculateBoundingClientRect();
```

Comment:
My suggestion here is to remove this as well as the `setTimeout` right below and instead trigger an initial value on stream that is observing clientBoundingRect changes. We could use `startWith` for that. Move the initial call of `calculateBoundingClientRect` into `ngOnInit`. This feels a little bit cleaner to me and also removes the need for the `setTimeout`.

So `ngOnInit` would look something like this:

```
this.calculateBoundingClientRect();
this.observeBoundingRectChanges();
this.observeSelectableItems();
...
```

And `observeBoundingRectChanges` can be refactored to the following:
 
```
private observeBoundingRectChanges() {
  this.ngZone.runOutsideAngular(() => {
    const resize$ = fromEvent(window, 'resize');
    const windowScroll$ = fromEvent(window, 'scroll');
    const containerScroll$ = fromEvent(this.host, 'scroll');

    merge(resize$, windowScroll$, containerScroll$)
      .pipe(
        startWith('INITIAL_UPDATE'),
        auditTime(AUDIT_TIME),
        takeUntil(this.destroy$)
      ).subscribe(() => this.update());
  });
}
```

What do you think?

This will also remove the need for the `setTimeout` in the `select-item.directive` making it less "hacky".